performance-tests/bench_2/builder_idl/Builder.idl

```diff
@@ -297,6 +297,7 @@ module Builder {
   struct DataReaderConfig {
     string name;
     string topic_name;
+    string cft_name;
```

Comment:
I think I would prefer to use just the single topic_name and sort out where it belongs internally. I realize this will make the lookup a bit more complicated, but after taking another look at the spec, it seems like all the topic classes (normal, content filtered, multitopic) inherit from TopicDescription which holds info about the participant, name, and type name. This makes me think a function like "lookup_topicdescription" on the DDS participant is secretly dictating that normal topics, content filtered topics, and multitopics for the same participant can't share a name... and so I'd rather have our API reflect that.

Also worth noting that the multitopic class (which we may one day support in bench) doesn't differentiate between normal and content filtered topics in its create method, it just takes an SQL-like expression that lists topic names as strings... we'll have to decide internally if they're normal or content filtered.